addon/components/nypr-brick-row-blowout/column.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
+import layout from '../../templates/components/nypr-brick-row-blowout/column';
+
+export default Ember.Component.extend({
+  layout,
+  classNames: ['row__col'],
+  classNameBindings: ['flipped']
```

Comment:
Looks like there may be a small attribute name inconsistency between flip vs. flipped - on this component, if you pass `flipped=true` it would get flipped, but down on `nypr-brick-row-blowout.hbs` (https://github.com/nypublicradio/nypr-ui/pull/39/files#diff-6141847a1a5efc27761d45f7bd66729fR9), it checks for `flip=true`.

I think if this line was written as `classNameBindings: ['flip:flipped']` they would both simply require `flip=true`